bin.src/makeCoaddTempExp.py

```diff
@@ -20,9 +20,5 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-import lsst.pex.logging as pexLog
 from lsst.pipe.tasks.makeCoaddTempExp import MakeCoaddTempExpTask
-
-pexLog.Trace.setVerbosity('lsst.coadd', 3)
-pexLog.Trace.setVerbosity('lsst.ip.diffim', 1)
```

Comment:
@r-owen  , I removed these because I thought they were accidentally left here. Are they here intentionally?  If so I will add them back in the `lsst.log` form.